lib/components/primary_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_movie_clean/gen/colors.gen.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class PrimaryButton extends StatelessWidget {
+  const PrimaryButton(
+      {this.width,
+      this.height,
+      this.onPressed,
+      this.title,
+      this.textStyle,
+      super.key});
```

Comment:
add comma at last param then format code. It looks better
```suggestion
      super.key,});
```